superbench/runner/runner.py

```diff
@@ -121,7 +128,35 @@ def run(self):
                 continue
             benchmark_config = self._sb_benchmarks[benchmark_name]
             for mode in benchmark_config.modes or []:
-                if mode.name == 'torch.distributed':
+                if mode.name == 'local':
+                    logger.info('Runner is going to run %s.', benchmark_name)
+
+                    def run_proc(vars, rets):
+                        mode.update(vars)
+                        rc = self._ansible_client.run(
+                            self._ansible_client.get_shell_config(
+                                runner_command.format(
+                                    self.__get_mode_command(
+                                        mode, ('sb exec -c sb.config.yaml -C '
+                                               'superbench.enable={name}').format(name=benchmark_name)
+                                    )
+                                ),
+                                sudo=True
+                            )
+                        )
+                        rets[vars['proc_rank']] = rc
+
+                    jobs = []
+                    rets = multiprocessing.Manager().dict()
+                    for proc_rank in range(mode.proc_num or 1):
```

Comment:
What happens if mode.proc_num is -1? Or 1.2?

Reply:
it may fail, haven't added config validation yet
https://github.com/microsoft/superbenchmark/blob/5e9f948df21422f119b6d7ef097ba7265664055e/superbench/executor/executor.py#L45-L51

Reply:
Maybe we should handle it somewhere and we don't use `mode.proc_num or 1` to handle this scenario.

Reply:
updated